storage-resize-images/README.md

```diff
@@ -37,10 +37,7 @@ To install an extension, your project must be on the [Blaze (pay as you go) plan
  - Cloud Storage
  - Cloud Functions (Node.js 10+ runtime. [See FAQs](https://firebase.google.com/support/faq#expandable-24))
 
-
-
-
-**Configuration Parameters:**
+#### Configuration Parameters
```

Comment:
I think the `generate-readmes` script creates asterisks - since this file is auto-generated, let's not stray from the scripts